src/main/webapp/jsp/instructorCourseEnroll.jsp

```diff
@@ -1,43 +1,59 @@
 <%@ page trimDirectiveWhitespaces="true" %>
 <%@ page pageEncoding="UTF-8" %>
 <%@ page import="teammates.common.util.Const" %>
+<%@ page import="teammates.common.util.FrontEndLibrary" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
 <%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
 <%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
 
+<c:set var="cssIncludes">
+  <link rel="stylesheet" href="<%= FrontEndLibrary.HANDSONTABLE_CSS %>" type="text/css">
+</c:set>
 <c:set var="jsIncludes">
+  <script type="text/javascript" src="<%= FrontEndLibrary.HANDSONTABLE %>"></script>
   <script type="text/javascript" src="/js/instructorCourseEnrollPage.js"></script>
 </c:set>
 <c:set var="SESSION_TOKEN">
   <%=Const.ParamsNames.SESSION_TOKEN%>
 </c:set>
+<ti:instructorPage title="Enroll Students for ${data.courseId}" cssIncludes="${cssIncludes}" jsIncludes="${jsIncludes}">
 
-<ti:instructorPage title="Enroll Students for ${data.courseId}" jsIncludes="${jsIncludes}">
-  <div class="instructionImg">
-    <img src="/images/enrollInstructions.gif" class="img-responsive" border="0" >
-  </div>
-  <br>
   <div class="panel panel-primary">
     <div class="panel-body fill-plain">
-      <div class="text-muted">
-        <span class="glyphicon glyphicon-exclamation-sign glyphicon-primary"></span> If you want to enroll more then <strong>100</strong> students into one course, divide students into sections containing no more than <strong>100</strong> students.
+      <div class="text-muted padding-15px">
+        <span class="glyphicon glyphicon-exclamation-sign glyphicon-primary"></span> If you want to enroll more than
+        <strong>100</strong> students into one course, divide students into sections containing no more than
+        <strong>100</strong> students.
       </div>
-      <br>
-      <form action="${data.instructorCourseEnrollSaveLink}" method="post" class="form-horizontal" role="form">
+      <form id="student-data-spreadsheet-form" action="${data.instructorCourseEnrollSaveLink}" method="post"
+            class="form-horizontal" role="form">
         <input type="hidden" name="${SESSION_TOKEN}" value="${data.sessionToken}">
         <div class="col-md-12">
           <div class="form-group">
-            <label for="instructions" class="col-sm-1 control-label">Student data:</label>
-            <div class="col-sm-11">
-              <textarea class="form-control" id="enrollstudents" name="enrollstudents" rows="6" cols="120" style="max-width:100%;" placeholder="Paste student data here ...">${fn:escapeXml(data.enrollStudents)}</textarea>
+            <div id="student-data-spreadsheet">
+              <div class="panel-body">
+                <div id="spreadsheet"></div>
+              </div>
+              <t:statusMessage statusMessagesToUser="${data.statusMessagesToUser}"/>
```

Comment:
Doesn't look right to put this (the status message) here. It should be before the `student-data-spreadsheet` div?

Reply:
Here's the new look.

![image](https://user-images.githubusercontent.com/25320782/40766153-1b0a90f6-64e1-11e8-8603-69a9bf0ef589.png)